_posts/2020-09-08-communication-backend-comparison.md

```diff
@@ -0,0 +1,114 @@
+---
+layout: post
+title: Communication Backends, Raw performance benchmarking
+author: e_hoelzl
+published: true
+tags: [performance, results]
+excerpt_separator: <!--more-->
+---
+
+Distributed learning requires workers to collaborate by sharing learned information with their "colleagues". As MLBench
+supports both one and multiple processes per node, in addition to multi-node training, communication between workers is crucial
+and will heavily affect performance, notably for compute bound training algorithms.
+
+This Blog post addresses and analyzes the raw performance of different communication backends, used to transmit tensors and other
+information between the workers.
```

Comment:
..backends `on commodity communication hardware`.  and maybe say `large arrays or tensors`. no need for sending other info, no?